src/FormCollection/README.md

```diff
@@ -0,0 +1,190 @@
+# UX Form Collection
+
+Symfony UX Form collection is a Symfony bundle providing light UX for collection
+in Symfony Forms.
+
+## Installation
+
+UX Form Collection requires PHP 7.2+ and Symfony 4.4+.
+
+Install this bundle using Composer and Symfony Flex:
+
+```sh
+composer require symfony/ux-form-collection
+
+# Don't forget to install the JavaScript dependencies as well and compile
+yarn install --force
+yarn encore dev
+```
+
+Also make sure you have at least version 2.0 of [@symfony/stimulus-bridge](https://github.com/symfony/stimulus-bridge)
+in your `package.json` file.
+
+## Use predefined theme
+
+You need to select the right theme from the one you are using:
+
+```yaml
+# config/packages/twig.yaml
+twig:
+    # For bootstrap for example
+    form_themes: ['@FormCollection/form_theme_div.html.twig']
+```
+
+There are 2 predefined themes available:
+
+-   `@FormCollection/form_theme_div.html.twig`
+-   `@FormCollection/form_theme_table.html.twig`
+
+[Check the Symfony doc](https://symfony.com/doc/4.4/form/form_themes.html) for the different ways to set themes in Symfony.
+
+## Use a custom form theme
```

Comment:
Let's move this section down, below `Usage`. And we can call it: `Customizing how the Field Renders`.